docs/database.rst

```diff
@@ -1,47 +1,84 @@
-  filesystem (performance).
+From Mailu 1.9, Mailu has a :ref:`cli command (link) <config-export>` for exporting and importing the complete Mailu configuration.
+Using this tool it is very easy to switch what database back-end is used for Mailu.
+Unfortunately roundcube does not have a tool for exporting/importing roundcube configuration.
+This means it is not possible to switch the database back-end used by roundcube using out of box tools.
 
-To make use of this functionality, just select `postgresql` as database flavor.
-Don't select the usage of an external database. The ``docker-compose.yml`` and ``mailu.env``
-will pull in ``mailu/postgresql``. This image and ``mailu/admin`` contain all the scripts
-to automatically setup the database.
+To switch to a different database back-end:
 
-After bring up the service, it might be useful to check the logs with:
+1. Run config-export to export the configuration. E.g.  `docker-compose exec admin flask mailu config-export --secrets --output mail-config.yml`
+2. Set up your new database server. Refer to the subsequent sections for tips for creating the database.
+3. Modify the database settings (DB_*) in mailu.env. Refer to the :ref:`configuration guide (link) <db_settings>` for the exact settings.
+4. Start your Mailu deployment.
+5. Run config-import to import the configuration. E.g. `docker exec -i $(docker-compose ps -q admin) flask mailu config-import -v < mail-config.yml`
+
+Mailu has now been switched to the new database back-end. The Mailu configuration has also been migrated.
+
+.. note::
+   The setup configuration wizard (setup.mailu.io) only supports creating config files for the same database back-end. When creating new config files, select the desired database flavour in the setup and enter dummy values for roundcube.
+   In the generated mailu.env file, configure all ROUNDCUBE_DB_* environment variables to the old values. For SQLite you can remove all the ROUNDCUBE_DB_* values.
 
-.. code-block:: bash
 
-  docker-compose logs -f admin database
+External MySQL/MariaDB
+----------------------
+
+It is also possible to use a MySQL/MariaDB database server, hosted elsewhere.
+In this case you'll have to take to create an empty database for Mailu, corresponding user,
+password and sufficient privileges on the database to ``CREATE TABLE``, ``DROP`` etc.
+Usually making the user owner of the database would be the easiest thing to do.
+
+The following commands can serve as an example on how to set up MySQL/MariaDB for Mailu usage.
+Adjust this to your own liking.
+
+.. code-block:: sql
 
-External Postgresql
+  mysql> CREATE DATABASE mailu;
+  mysql> CREATE USER 'mailu'@'%' IDENTIFIED BY 'my-strong-password-here';
+  mysql> GRANT ALL PRIVILEGES ON mailu.* TO 'mailu'@'%';
+  mysql> FLUSH PRIVILEGES;
+
+Note that if you get any errors related to ``caching_sha2_password`` it can be solved by changing the encryption
```

Comment:
I'd get rid of that paragraph altogether with the change above

Reply:
Ok. I will make the above change and will remove this paragraph.